index.html

```diff
@@ -3,9 +3,35 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins&display=swap">
+    <link rel="stylesheet" href="./fonts/CreteRound-Italic.ttf">
     <title>Mobile Portfolio</title>
 </head>
 <body>
-    <h1>My Portfolio</h1>
+    <header class="top-navigation">
+        <h1>My Logo</h1>
+        <div class="menu-bar">
+            <img src="./images/Enabled.png" alt="">
+        </div>
+    </header>
+    <section>
+      <div class="bio-container">
+        <div class="info-holder">
+          <h1>Hey There. <br>I'm Prince.</h1>
+          <h3>I'm a software Developer, </h3>
+          <p>I can help you build a product, feature or website look through some of my work
+            and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+          </p>
+        </div>
+        <div class="icon-container">
+            <a href="#"><li><img src="./images/icon1.png" alt=""></li></a>
+            <a href="#"><li><img src="./images/icon2.png" alt=""></li></a>
+            <a href="#"><li><img src="./images/icon3.png" alt=""></li></a>
+            <a href="#"><li><img src="./images/icon4.png" alt=""></li></a>
+            <a href="#"><li><img src="./images/icon5.png" alt=""></li></a>
+        </div>
```

Comment:
- Kindly consider listing these social media icons with a ul tag and li tag for each and also a link. This way a user can easily click on each and will be redirected to the social media page of that particular icon.
- Kindly also consider using Flexbox because it gives us a lot of options to align all our elements, plus, it's responsive.